base_comment_template/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,11 @@
+Go to *Settings > Technical > Reporting > Comment Templates* and start designing you comment templates.
+
+The template are general, and can be attached to any Model and based on some domain defined in the template.
```

Comment:
Why select any model?
I believe there are a lot of models that can't use this templates

Reply:
I added fields view get to filter models that have reports.